pkg/build/docker_go.go

```diff
@@ -424,4 +431,6 @@ COPY --from=builder /plan/testplan /
 
 EXPOSE 6060
 ENTRYPOINT [ "/testplan"]
+
+{{ end }}
```

Comment:
I think this should be above `EXPOSE 6060`.

Reply:
You also need:
```
{{ if .SkipRuntimeImage }}
RUN cp /testground_dep_list /
RUN /plan/testplan /
{{ end }}
```

Reply:
Oh yeah, absolutely. And we need to put the test plan binary at the root.

Reply:
Achieved the symmetry but differently: 

[https://github.com/testground/testground/pull/1078/commits/548da321a79cc807d1689fc69f1beec079873371 ](https://github.com/testground/testground/pull/1078/commits/548da321a79cc807d1689fc69f1beec079873371)